plugin/mouse/mouse.py

```diff
@@ -111,43 +109,11 @@ def mouse_hide_cursor():
 
     def mouse_wake():
         """Enable control mouse, zoom mouse, and disables cursor"""
-        eye_zoom_mouse.toggle_zoom_mouse(True)
+        actions.tracking.control_zoom_toggle()
```

Comment:
Might want to make this:
```python
actions.tracking.control_zoom_toggle(True)
```

At present it looks like it would turn off zoom mouse if invoked twice.

Reply:
> Looks fine (though don't have an eye tracker to test). Couple of minor suggestions.

I've addressed your feedback, thanks.  I have an eye tracker and have tested these changes.